include/macros.h

```diff
@@ -98,6 +98,28 @@
 
 #define SPRITE_PIXEL_SCALE (5.0 / 7.0)
 
+#define VAR_PROJECTILE_HITBOX_STATE varTable[0]
```

Comment:
I think these would be better in evt.h or something instead of the general macros.h file